.travis.yml

```diff
@@ -2,7 +2,7 @@ sudo: false
 
 env:
   global:
-    - CONDA_DEPS="pip flake8 numpy scipy numba joblib dask" PIP_DEPS="pytest coveralls pytest-cov"
+    - CONDA_DEPS="pip flake8 numpy scipy numba joblib dask nose" PIP_DEPS="pytest coveralls pytest-cov"
```

Comment:
Travis told me. If you look at the [output](https://travis-ci.org/uwescience/pulse2percept/builds/178494226) of testing 3ab0fabe7d61cc12298c40fe6cf6a73b872b589b, it says at the bottom: `ImportError: Need nose >= 1.0.0 for tests - see http://somethingaboutorange.com/mrl/projects/nose`. It looks like it has to do with `npt.assert_raises`. Once I included nose, the error went away.